.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push]
+
+jobs:
+  lint-and-test:
+    name: Lint and test package on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
```

Comment:
Why is it Java version 12? Java 11 is the latest LTS.

Reply:
Also recommend using the LTS version 11.